src/pages/StatePartner Dashboard/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useSelector } from "react-redux";
+import { METHODS } from "../../services/api";
+import { Link } from "react-router-dom";
+import { PATHS } from "../../constant";
+import { hasOneFrom } from "../../common/utils";
+import { Redirect } from "react-router";
+import "./style.scss";
+
+function StatePartnerDashboard() {
+  const user = useSelector(({ User }) => User);
+  const [districtPartner, setDistrictPartner] = useState([]);
+  const [districtPartnerData, setDistrictPartnerData] = useState([]);
+  const [filterData, setFilterData] = useState([]);
+  const [value, setValue] = useState();
+
+  useEffect(() => {
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}/partners/23/groups`,
+      headers: {
+        accept: "application/json",
+        Authorization: user.data.token,
+      },
+    }).then((res) => {
+      setDistrictPartner(res.data);
+      setDistrictPartnerData(res.data.partner_groups_data);
+      setFilterData([...filterData, res.data.partner_groups_data[0]]);
+    });
+  }, []);
+
+  const handleSelcet = (e) => {
+    const stateFilterData = districtPartnerData.filter(
+      (el) => el.partner_group_name == e.target.value
```

Comment:
Using `===` (or `!==`) instead of `==` (and `!=`) is recommended here and anywhere you don't need to do type conversion before the comparison (i.e., if what appears on the left and right of the `===` are expected to be the same type such as here where they should both be strings).
```suggestion
      (el) => el.partner_group_name === e.target.value
```